File: SAM Issue 3/updated SAM files/Java/Q02aCorrect.java
import java.util.Scanner;
public class Q02a 
{
    public static void main (String[] args)
	{
        Scanner input = new Scanner(System.in);
		
		// Print prompt and take country from user			
        System.out.print("Enter the country you are visiting from: ");
        String country = input.next();
		
		// Tell the user their country		
        System.out.println("You are from: " + country);
		
		// Take number of adults in party from user			
        System.out.print("Enter the number of adults in your party: ");
        int adults = input.nextInt();
		
		// Take number of children in party from user		
        System.out.print("Enter the number of children in your party: ");
        int children = input.nextInt();        
		
		// Calculate total number in party		
        int total = adults + children;
		
		// Tell the user the total number of people in their party		
        System.out.println("The total in your party is: " + total);    

    }   
}

File: SAM Issue 3/updated SAM files/C#/Q02aCorrect.cs
using System;
namespace Q02a
{
    class Q02a
    {
        static void Main(String[] args)
        {	
			// Print prompt and take country from user			
            Console.WriteLine("Enter the country you are visiting from: ");
            String country = Console.ReadLine();
            Console.WriteLine("You are from: " + country);
			
			// Tell the user their country			
            Console.WriteLine("Enter the number of adults in your party: ");
			
			// Take number of adults in party from user
            int adults = Convert.ToInt32(Console.ReadLine());
            Console.WriteLine("Enter the number of children in your party: ");
			
			// Take number of children in party from user			
            int children = Convert.ToInt32(Console.ReadLine());
			
			// Calculate total number in party			
            int total = adults + children;
			
			// Tell the user the total number of people in their party			
            Console.WriteLine("The total in your party is: " + total);
			
			Console.ReadKey();
        }
    }
}
